framework/ui/src/components/organisms/vc-blade/_internal/vc-blade-toolbar/vc-blade-toolbar.vue

```diff
@@ -41,10 +41,24 @@ export default defineComponent({
 
   setup(props) {
     const isExpanded = ref(true);
+    try {
+      isExpanded.value =
+        localStorage.getItem("VC_BLADE_TOOLBAR_IS_EXPANDED") === "true";
+    } catch (err) {
+      isExpanded.value = false;
```

Comment:
isExpanded is initialized as "true" on line 43, it's confusing setting it to false here. Default value should be "true"